web/src/components/Download.vue

```diff
@@ -31,7 +31,7 @@ export default {
     dataset() { return this.$store.getters.dataset(this.id); },
     dataframe() { return this.dataset.validatedMeasurements; },
     ready() { return this.$store.getters.ready(this.id); },
-    loading() { return this.$store.state.loading; },
+    loading() { return this.$store.getters.waiting; },
```

Comment:
could you make this `waiting` also?   `loading` is overloaded and should not have different meanings between files.